dev/README_RELEASE_AIRFLOW.md

```diff
@@ -757,16 +783,17 @@ previously released RC candidates in "${AIRFLOW_SOURCES}/dist":
 - Upload the package to PyPI's production environment:
 
     ```shell script
-    twine upload -r pypi dist/*
+    twine upload -r pypi *.whl *${VERSION}.tar.gz
```

Comment:
```suggestion
    twine upload -r pypi *${VERSION}*.whl *${VERSION}.tar.gz
```

To be safe, wdyt?

Reply:
It's in ${VERSION} dir already. We should not have more

Reply:
It'd have to be this:

```suggestion
    twine upload -r pypi *${VERSION}-py3-none-any.whl *${VERSION}.tar.gz
```

At that point, might even be worth removing globbing:

```suggestion
    twine upload -r pypi apache_airflow-${VERSION}-py3-none-any.whl apache-airflow-${VERSION}.tar.gz
```

No strong preference tbh.

Reply:
Oh yeah, Jarek is right..

The older code, pushed it from dist/ dir... This is fine

Reply:
It once happened with me that I had elasticsearch provider under dist dir.. My scars from having burnt from there just got better of me 😃

Reply:
Haha yeah, my fear of that was the driver to upload from svn instead 👍